Functions/xASL_bids_BIDS2Legacy.m

```diff
@@ -32,7 +33,8 @@
 % 9. Parse M0
 % 10. Create DataPar.json
 % 11. Copy participants.tsv
-% 12. Clean up
+% 12. Add dataset_description.json
```

Comment:
Theoretically, enough to have it there. But practically, we mostly need it on the subject level - written to all ASL and M0 JSONs, because at the individual file level, we will always check what ExploreASL version was used.

Our motivation here is that we currently remove and apply the Philips scalings already when doing DCM2BIDS conversion. But we leave the SIemens and GE scalings only in the quantification module, because they are not really stored directly in DICOM. But for ASL-BIDS, we wanted to remove any scalings, independent of what is their origin. So once we implement that, we want to be sure what version of ExploreASL was used to create data - data generated by the old version will apply these scalings still in quantification, data generated by the newer version will apply the scalings already in import - as promised in ASL-BIDS.

Reply:
Sounds good. It's just added to all JSONs now in e7ee311 👍 
I can remove it from the dataPar.json if you want, but it probably doesn't hurt either.

Reply:
That's an elegant way to add it. We indeed want it in all JSON data sidecars. We want it in `dataset_description`, it doesn't hurt in `dataPar.json`. We probably don't want it in `participants.json` as this should have fields describing columns and an extra field that doesn't match the structure can be confusing. 
See the [BIDS-modality-agnostic-files.html](https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html). 

What do you think?

Reply:
Agree, just read about it. I exclude it in 3afe366  👍